draft-ietf-quic-transport.md

```diff
@@ -1532,6 +1532,24 @@ be unambiguously different to ensure no confusion about their interpretation.
 One way that a new format could be introduced is to define a TLS extension with
 a different codepoint.
 
+### Handshake Completion {#handshake-completion}
+
+The connection establishment handshake concludes when both parties have
```

Comment:
I don't agree with some of this text, but importantly, it's more than is needed. Suggested rephrase of this paragraph, and combine this with the next paragraph: "Connection handshake is complete when version negotiation is complete and 1-RTT keys are available at both endpoints."